terraform/root.tf

```diff
@@ -63,17 +63,17 @@ module "jenkins_backup_s3" {
   common_tags = local.common_tags
 }
 
-module "sonatype_intg" {
-  source      = "./modules/sonatype-build-task"
+module "s3_publish_intg" {
```

Comment:
Same question as here: https://github.com/nationalarchives/tdr-terraform-backend/pull/38/files#r403910570

Reply:
Yeah, fair point. I've changed it to just create the one.